angular-apps/main-categories-app/src/app/app.component.html

```diff
@@ -0,0 +1,22 @@
+<section class="categories">
+    <div class="container">
+      <div class="row">
+          <div class="categories__slider">
+            <owl-carousel-o [options]="customOptions">
+                <ng-container *ngFor="let category of categories">
+                <ng-template class="slide" carouselSlide [id]="'m' + category.id">
+                    <div class="col-lg-3">
+                        <div
+                            class="categories__item set-bg"
+                            [style.background-image]="'url(http://localhost:5000/' + category.image_url + ')'"
```

Comment:
I think you should use the configuration or environment for the URL localhost:5000.